bin/pd2pg

```diff
@@ -272,6 +276,7 @@ class PG2PD
         )
         data = JSON.parse(response.body)
         total = data["total"]
+        more = data["more"]
```

Comment:
I'm slightly worried about this getting stuck in a loop, some kind of sanity check to ensure that the pages we are requesting are always increasing would be nice I think

Reply:
... or just initialize more to false inside the loop, so it only pages when `more` exists on the response and is truthy.